Makefile

```diff
@@ -12,7 +12,6 @@ DOCKER_WORKDIR=/proj
 DOCKER_RUN=docker run --rm -e LD_FLAGS="$$LD_FLAGS" -v "$$PWD/.:${DOCKER_WORKDIR}" -v "`go env GOPATH`/pkg/mod/.:/go/pkg/mod:ro" -w ${DOCKER_WORKDIR}
```

Comment:
@eternal-flame-AD Do you have time to give this PR a small manual test? It should work as the e2e tests are successful, but there may be something I overlooked.

The UI coloring changed a little because the defaults of the material-ui lib changed.

Reply:
Sure, will do today or tomorrow!